cmd/lotus-storage-miner/init_service.go

```diff
@@ -63,12 +39,10 @@ var serviceCmd = &cli.Command{
 			Name:  "nosync",
 			Usage: "don't check full-node sync status",
 		},
-
-		&cli.BoolFlag{
-			Name:  "enable-market",
-			Usage: "enable market module",
+		&cli.StringSliceFlag{
+			Name:  "name",
```

Comment:
Maybe `type`? `lotus-miner init service --name markets ...` looks a bit weild

Reply:
Renamed to `type`.